packages/components/docs/QueryModel.md

```diff
@@ -0,0 +1,164 @@
+# QueryModel API
+
+At a high level the QueryModel API is a wrapper around the selectRows API. If you need to retrieve data from a LabKey
+table, so you can render it in a React component, then the QueryModel API is most likely what you want.
+
+### QueryModel / QueryConfig
+
+This is the base model used to store all the data for a query. This stores some client-side only data as well as data
+retrieved by the server. You can manually instantiate a `QueryModel`, but you will almost never do this, instead you will
+use the `withQueryModels` HOC to inject the needed QueryModel(s) into your component. To create a QueryModel you will need
+to define a `QueryConfig` object. At a minimum, your `QueryConfig` must have a valid SchemaQuery, but we also support
+many other attributes that allow you to configure the model before it is loaded, all of the attributes can be found
+on the `QueryConfig` interface at `src/QueryModel/QueryModel.ts`.
```

Comment:
should we link this to the github pages?
https://labkey.github.io/labkey-ui-components/interfaces/queryconfig.html

Reply:
@labkey-alan I saw that you did add this link. Looking back at it now, there are probably a few other places in this doc we could drop in that link to gh pages as well. I think there are a couple other places you references file paths or interaces.